README.md

```diff
@@ -1 +1,243 @@
 For everything you need to know about the seed project, including a step-by-step process, check out the documentation on our [website](https://documentation.chartiq.com/finsemble/tutorial-gettingStarted.html).
+
+<!-- TODO: Update tutorials to point to _tutorials_ folder at the top level -->
+<!-- TODO: Move creating of components to tutorials and have URL to point to CSS -->
+
+# Finsemble Seed Project
+
+This article lists the contents of the Finsemble seed project, how the seed project can be extended, and some information about upgrading from the previous seed project structure.
+
+- [Project structure](#project-structure)
+- [Extending gulpfile](#extending-gulpfile)
+- [Extending server functionality](#extending-server-functionality)
+- [Upgrade instructions](#upgrade-instructions)
+
+## Project structure
+
+The Finsemble seed project provides a basic structure to help developers get up and running as quickly as possible. The seed project provides the basic skeleton for a Finsemble application that can be extended to suit your organization's needs. It also includes some functionality to make development faster and easier, like a basic build process and hot reload.
+
+- _gulpfile.js_ - The main gulpfile for the project that includes the basic tasks used to build and run a Finsemble application for development.
+- _gulpfile-extension.js_ (optional) - File that can be used to add/modify the functionality of the gulpfile. This file isn't included in the base seed project to prevent conflicts when upgrading the base project. See [Extending Gulpfile](#extending-gulpfile) for more information.
+- _build/webpack_ - Includes all of the files used by the seed project to build the application.
+    - _webpack.default.files.entries.json_ - This specifies the entry and output files for the files built for a default Finsemble application.
+    - _webpack.files.entries.json_ - This file is where developer-added files should be listed. This file is empty in the base Finsemble seed project to prevent merge conflicts when updating the seed project.
+- _configs/application_ - This folder contains all of the base configuration for the Finsemble application. _component.json_, _config.json_ and _services.json_ are empty and developer-added configuration should go here. The files in this folder are merged together to build the application configuration. This configuration can be changed at run time using [Dynamic Configuration](https://documentation.chartiq.com/finsemble/tutorial-dynamicConfiguration.html).
+- _configs/openfin_ - Contains the [OpenFin application config](https://openfin.co/documentation/application-config/) used to start up the Finsemble application. The default manifest for development is included, and additional configurations can be placed in this folder.
```

Comment:
Might be worth pointing out that this is also what openfin means when they talk about the application manifest.

Reply:
Does this sound good:
> Contains the [OpenFin application config](https://openfin.co/documentation/application-config/) (also known as the application manifest) used ...

Reply:
perfect